### PR TITLE
[rename network] Rename network names for Testnet & Devnet

### PR DESCRIPTION
**Pull request by janet-harmony**
## Issue

`-N testnet` for Testnet/Pangaea
`-N devnet` for Devnet

### Unit Test Coverage

Before:

```
<!-- copy/paste 'go test -cover' result in the directory you made change -->
```

After:

```
<!-- copy/paste 'go test -cover' result in the directory you made change -->
```

## Operational Checklist

1. **Does this PR introduce backward-incompatible changes to the on-disk data structure and/or the over-the-wire protocol?**. (If no, skip to question 8.)

    **YES|NO**

2. **Describe the migration plan.**. For each flag epoch, describe what changes take place at the flag epoch, the anticipated interactions between upgraded/non-upgraded nodes, and any special operational considerations for the migration.

3. **Describe how the plan was tested.**

4. **How much minimum baking period after the last flag epoch should we allow on Pangaea before promotion onto mainnet?**

5. **What are the planned flag epoch numbers and their ETAs on Pangaea?**

6. **What are the planned flag epoch numbers and their ETAs on mainnet?**

    Note that this must be enough to cover baking period on Pangaea.

7. **What should node operators know about this planned change?**

8. **Does this PR introduce backward-incompatible changes *NOT* related to on-disk data structure and/or over-the-wire protocol?** (If no, continue to question 11.)

    **YES|NO**

9. **Does the existing `node.sh` continue to work with this change?**
Changes to the node.sh

10. **What should node operators know about this change?**

11. **Does this PR introduce significant changes to the operational requirements of the node software, such as >20% increase in CPU, memory, and/or disk usage?**

### CODE REVIEW

File: scripts/node.sh [hunk 1]
```diff
@@ -222,27 +222,27 @@ main)
...
   ;;
-dev)
+dev|devnet)
   bootnodes=(
     /ip4/52.40.84.2/tcp/9870/p2p/QmZJJx6AdaoEkGLrYG4JeLCKeCKDjnFz2wfHNHxAqFSGA9
     /ip4/54.86.126.90/tcp/9870/p2p/Qmdfjtk6hPoyrH1zVD9PEH4zfWLo38dP2mDvvKXfh3tnEv
   )
   REL=devnet
-  network_type=pangaea
+  network_type=devnet
```
Comment (LeoHChen) [inline scripts/node.sh]: don't change this one.

File: scripts/node.sh [hunk 2]
```diff
@@ -106,8 +106,8 @@ usage: ${progname} [-1ch] [-k KEYFILE]
    -d             just download the Harmony binaries (default: off)
    -D             do not download Harmony binaries (default: download when start)
    -m             collect and upload node metrics to harmony prometheus + grafana
-   -N network     join the given network (main, beta, pangaea, dev; default: main)
-   -t             equivalent to -N pangaea (deprecated)
+   -N network     join the given network (main, beta, testnet, devnet, dev; default: main)
```
Comment (LeoHChen) [inline scripts/node.sh]: remove beta, please.

File: scripts/node.sh [hunk 3]
```diff
@@ -106,8 +106,8 @@ usage: ${progname} [-1ch] [-k KEYFILE]
    -d             just download the Harmony binaries (default: off)
    -D             do not download Harmony binaries (default: download when start)
    -m             collect and upload node metrics to harmony prometheus + grafana
-   -N network     join the given network (main, beta, pangaea, dev; default: main)
-   -t             equivalent to -N pangaea (deprecated)
+   -N network     join the given network (main, testnet, devnet, dev; default: main)
```
Comment (rlan35) [inline scripts/node.sh]: let's do "main, test, dev; default: main"

Comment (rlan35) [reply scripts/node.sh]: Please change main to mainnet too. Make it consistent.

**Pull request merged**